boards/riscv/beagle_v_starlight/board.cmake

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
as per @pdp7's recommendation, I would change to `beaglev_starlight_jh7100`

Reply:
Fixed.